src/raygun.js

```diff
@@ -646,6 +646,10 @@ var raygunFactory = function (window, $, Raygun, undefined) {
             }
         }
 
+        if (Raygun.Utilities.isReactNative() && !Raygun.Utilities.contains(options.tags), 'React Native') {
```

Comment:
Noticed this because I momentarily used the v2.6.0-SNAPSHOT.4 release in my app and I kept getting 'React Native' as a tag in the reports.